docs/docs/50-running/03-certificates.md

```diff
@@ -53,3 +53,32 @@ acorn run -p my-app.example.com:web [MY_APP_IMAGE]
 
 Acorn will automatically inspect each certificate in the Acorn namespace for one that can be used with `my-app.example.com`.
 If no TLS secret is found with that FQDN, it will be exposed on HTTP only.
+
+### Issuing custom domain certs
+
+Acorn Let's Encrypt integration does not issue certificates for custom domain. Instead, you will rely on external cert-manager to issue certificates. To do so, you will need to create a cluster issuer first. For more information on how to configure cert-manager, see [cert-manager docs](https://cert-manager.io/docs/).
+
+```yaml 
```

Comment:
Docs has been called out on how to use it. Those setting will be managed in hub. https://github.com/acorn-io/infra-provisioning-cdk/pull/23